plugin/plugins/galgame_plugin/static/index.html

```
@@ -18,12 +18,12 @@
       <div class="onboarding-card">
         <div class="onboarding-header">
           <p class="eyebrow">N.E.K.O</p>
-          <h1>Galgame 游玩助手</h1>
-          <p class="onboarding-subtitle">让猫娘陪你一起玩 Galgame</p>
+          <h1 data-i18n="ui.app.title">Galgame 游玩助手</h1>
+          <p class="onboarding-subtitle" data-i18n="ui.app.subtitle">让猫娘陪你一起玩 Galgame</p>
         </div>
         <div id="onboardingSteps" class="onboarding-steps"></div>
         <div id="onboardingActions" class="onboarding-actions"></div>
-        <button class="ghost onboarding-skip" data-skip-onboarding>跳过引导，直接配置 →</button>
+        <button class="ghost onboarding-skip" data-skip-onboarding data-i18n="ui.app.skip_onboarding">跳过引导，直接配置 -></button>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)